### PR TITLE
Migrates Storage Class provisioner

### PR DESCRIPTION
**Pull request by bipuladh**
Signed-off-by: Bipul Adhikari <badhikar@redhat.com>

Recording:

https://user-images.githubusercontent.com/54092533/188412067-4e74a991-d782-4a02-8d12-f8b64f24d186.mp4

### CODE REVIEW

File: plugins/odf/console-extensions.json [hunk 1]
```diff
@@ -377,5 +377,176 @@
...
+          "csi.storage.k8s.io/controller-expand-secret-name": {
+            "name": "Expand Secret Name",
+            "hintText": "The namespace where provisioner secret is created",
+            "value": "grook-csi-cephfs-provisioner",
+            "visible": false
+          },
+          "csi.storage.k8s.io/controller-expand-secret-namespace": {
+            "name": "Expand Secret Namespace",
+            "hintText": "The namespace where provisioner secret is created",
+            "value": "gopenshift-storage",
```
Comment (SanjalKatiyar) [inline plugins/odf/console-extensions.json:545]:
```suggestion
            "value": "openshift-storage",
```

File: plugins/odf/console-extensions.json [hunk 2]
```diff
@@ -377,5 +377,176 @@
...
+          "csi.storage.k8s.io/node-stage-secret-namespace": {
+            "name": "Node Stage Secret Namespace",
+            "hintText": "The namespace where provisioner secret is created",
+            "value": "openshift-storage",
+            "visible": false
+          },
+          "csi.storage.k8s.io/controller-expand-secret-name": {
+            "name": "Expand Secret Name",
+            "hintText": "The namespace where provisioner secret is created",
+            "value": "grook-csi-cephfs-provisioner",
```
Comment (SanjalKatiyar) [inline plugins/odf/console-extensions.json:539]:
```suggestion
            "value": "rook-csi-cephfs-provisioner",
```

File: plugins/odf/console-extensions.json [hunk 3]
```diff
@@ -377,5 +377,176 @@
...
+          "csi.storage.k8s.io/provisioner-secret-namespace": {
+            "name": "Provisioner Secret Namespace",
+            "hintText": "The namespace where provisioner secret is created",
+            "value": "openshift-storage",
+            "visible": false
+          },
+          "csi.storage.k8s.io/node-stage-secret-name": {
+            "name": "Node Stage Secret Name",
+            "hintText": "The name of Node Stage secret",
+            "value": "grook-csi-cephfs-node",
```
Comment (SanjalKatiyar) [inline plugins/odf/console-extensions.json:527]:
```suggestion
            "value": "rook-csi-cephfs-node",
```

File: .prettierignore [hunk 4]
```diff
@@ -4,3 +4,4 @@ dist
 locales
 .husky
 .vscode
+*.scss
```
Comment (SanjalKatiyar) [inline .prettierignore:7]: it should be okay to format scss as well IMO... but not anything important...

Comment (bipuladh) [reply .prettierignore:7]: Rules for prettier and lint-css are clashing.

**Pull request merged**